app/static/css/style.css

```diff
@@ -0,0 +1,65 @@
+a.link-text {
+    text-decoration: none;
+    color: rgb(41, 158, 158);
+
+}
+
+a.link-text:hover {
+    color: rgb(16, 99, 99);
+}
+
+
+/* Authentication Forms */
+
+#registerForm {
+    align-items: center;
+    padding-top: 40px;
+    padding-bottom: 40px;
+}
+
+main.form-register,
+main.form-login {
+    display: block;
+}
+
+.card-wrapper {
+    display: flex;
+    justify-content: center;
+    min-width: 100%;
+    max-width: 800px;
+    height: auto;
+    padding: 40px 40px;
+    margin: auto;
+}
+
+.card {
+    margin-top: 80px;
+    -webkit-box-shadow: 0 10px 6px -6px #777;
+    -moz-box-shadow: 0 10px 6px -6px #777;
+    box-shadow: 0 10px 6px -6px #777;
+    border: .5px solid rgb(194, 194, 194);
```

Comment:
Half pixel unit of measurements should be avoided. They do not translate in Chrome, Opera, Edge. They do in Firefox/Safari but unless the user is on a DPI of 2.0 or 3.0 the unit will always be rendered as rounded instead. This could cause weird visual flucuation and should be avoided.

Reply:
Changed it to 1px.